content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,69 @@
+# Machine Learning Model Registry
+
+A **model registry** is a tool to catalog ML models and their versions. Models
+from your data science projects can be discovered, audited, shared, tested, and
+deployed from here. [Iterative](https://iterative.ai/) tools like
+[DVC](https://dvc.org/doc), [GTO], and [MLEM] enable these capabilities on top
+of Git, so you can stick to an existing software engineering stack. No more
+division between ML and App operations!
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries is between development and applications. See also [Data Registries]._
+
```

Comment:
Figure improvement idea: make Git a central concept e.g.

<img width="643" alt="image" src="https://user-images.githubusercontent.com/1477535/167967249-a47c99c8-fd42-48c1-84dc-695dd58fce8b.png">
> From GTO presentation

Reply:
p.s. applies to benefits list, which currently covers model registries in general, no Git-as-single-source-of-truth focus yet.